src/Component/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Todo } from '../../Type/Type';
+import { UserInfo } from '../UserInfo';
+import './Style.css';
+
+type Props = {
+  todos: Todo;
+};
```

Comment:
```suggestion
type Props = {
  todo: Todo;
};
```
Сюди передається тільки одне todo, а не декілька, а твоя назва говорить про множину